client/src/app/core/definitions/relations.ts

```diff
@@ -153,6 +153,40 @@ export function makeGenericO2O<V extends BaseViewModel, I>(args: {
     ];
 }
 
+export function makeGenericM2O<V extends BaseViewModel, I>(args: {
```

Comment:
here it is interesting which one is the O and which the M model, e.g. is the model with the content object (e.g. personal note) the M or O part? Please add a comment describing it.